### PR TITLE
distinct tidb only information function

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14467, https://github.com/pingcap/docs-cn/pull/14515
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/information-functions.md [hunk 1]
```diff
@@ -8,13 +8,12 @@ aliases: ['/docs/dev/functions-and-operators/information-functions/','/docs/dev/
 
 TiDB supports most of the [information functions](https://dev.mysql.com/doc/refman/5.7/en/information-functions.html) available in MySQL 5.7.
 
-## Supported functions
+## TiDB Supported MySQL functions
```
Comment (qiancai) [inline functions-and-operators/information-functions.md:11]:
```suggestion
## TiDB supported MySQL functions
```

File: functions-and-operators/information-functions.md [hunk 2]
```diff
@@ -26,6 +25,15 @@ TiDB supports most of the [information functions](https://dev.mysql.com/doc/refm
 | [`USER()`](https://dev.mysql.com/doc/refman/5.7/en/information-functions.html#function_user) | Return the user name and host name provided by the client    |
 | [`VERSION()`](https://dev.mysql.com/doc/refman/5.7/en/information-functions.html#function_version) | Return a string that indicates the MySQL server version   |
 
+## TiDB Specific functions
```
Comment (qiancai) [inline functions-and-operators/information-functions.md:28]:
```suggestion
## TiDB specific functions
```

File: functions-and-operators/information-functions.md [hunk 3]
```diff
@@ -26,6 +25,15 @@ TiDB supports most of the [information functions](https://dev.mysql.com/doc/refm
...
+
+Following functions are only supported by TiDB, there is no equivalent function in MySQL.
```
Comment (qiancai) [inline functions-and-operators/information-functions.md:30]:
```suggestion
The following function is only supported by TiDB, and there is no equivalent function in MySQL.
```

File: functions-and-operators/information-functions.md (hunk 1 above)
Comment (qiancai) [reply functions-and-operators/information-functions.md:11]: TiDB 英文文档里小节的名字统一用的是 sentence case

File: functions-and-operators/information-functions.md [hunk 4]
```diff
@@ -26,6 +25,15 @@ TiDB supports most of the [information functions](https://dev.mysql.com/doc/refm
...
 | [`VERSION()`](https://dev.mysql.com/doc/refman/5.7/en/information-functions.html#function_version) | Return a string that indicates the MySQL server version   |
 
+## TiDB specific functions
+
+The following function is only supported by TiDB, and there is no equivalent function in MySQL.
+
+| Name | Description |
+|:-----|:------------|
+| `CURRENT_RESOURCE_GROUP()`  | Return the name of the resource group that the current session is bound to |
+
```
Comment (hfxsd) [inline functions-and-operators/information-functions.md:35]:
```suggestion
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: functions-and-operators/tidb-functions.md [hunk 5]
```diff
@@ -21,6 +21,7 @@ The following functions are TiDB extensions, and are not present in MySQL:
 | `VITESS_HASH(str)` | The `VITESS_HASH` function returns the hash of a string that is compatible with Vitess' `HASH` function. This is intended to help the data migration from Vitess. |
 | `TIDB_SHARD()` | The `TIDB_SHARD` function can be used to create a shard index to scatter the index hotspot. A shard index is an expression index with a `TIDB_SHARD` function as the prefix.|
 | `TIDB_ROW_CHECKSUM()` | The `TIDB_ROW_CHECKSUM` function is used to query the checksum value of a row. This function can only be used in `SELECT` statements within the FastPlan process. That is, you can query through statements like `SELECT TIDB_ROW_CHECKSUM() FROM t WHERE id = ?` or `SELECT TIDB_ROW_CHECKSUM() FROM t WHERE id IN (?, ?, ...)`. See also: [Data integrity validation for single-row data](/ticdc/ticdc-integrity-check.md). |
+| `CURRENT_RESOURCE_GROUP()`  | The `CURRENT_RESOURCE_GROUP` function is used to return the resource group name that the current session is bound to. See also: [Use Resource Control to Achieve Resource Isolation](/tidb-resource-control.md) |
```
Comment (qiancai) [inline functions-and-operators/tidb-functions.md:24]:
```suggestion
| `CURRENT_RESOURCE_GROUP()`  | The `CURRENT_RESOURCE_GROUP` function is used to return the resource group name that the current session is bound to. See also: [Use Resource Control to Achieve Resource Isolation](/tidb-resource-control.md). |
```

**Comment by glorv** [comment]
@qiancai @hfxsd PTAL again, also add translation for #pingcap/docs-cn/pull/14515

File: functions-and-operators/tidb-functions.md [hunk 6]
```diff
@@ -337,3 +339,50 @@ The output is as follows:
 +----+------+------+---------------------+
 1 row in set (0.000 sec)
 ```
+
+### CURRENT_RESOURCE_GROUP
+
+The `CURRENT_RESOURCE_GROUP` function is used to show the resource group name the current session is bound to. When feature [Use Resource Control to Achieve Resource Isolation](/tidb-resource-control.md) is enabled, the available resources that can be used by SQL statements are restricted by the resource quota of the bound resource group.
+
+When a session is established, TiDB binds the session to the resource group that the login user is bound to by default. If the user is not bound to any resource groups, the session is bound to the `default` resource group. After the session is established, the bound resource group won't change by default, even if the user bound resource group is changed via [Modify the resource group bound to the user](/sql-statements/sql-statement-alter-user.md#modify-basic-user-information). You can use [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md) to change current session's bound resource group.
```
Comment (qiancai) [inline functions-and-operators/tidb-functions.md:347]:
```suggestion
When a session is established, TiDB binds the session to the resource group that the login user is bound to by default. If the user is not bound to any resource groups, the session is bound to the `default` resource group. Once the session is established, the bound resource group will not change by default, even if the bound resource group of the user is changed via [modifying the resource group bound to the user](/sql-statements/sql-statement-alter-user.md#modify-basic-user-information). To change the bound resource group of the current session, you can use [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md).
```

File: functions-and-operators/tidb-functions.md [hunk 7]
```diff
@@ -337,3 +339,50 @@ The output is as follows:
...
+```
++--------------------------+
+| CURRENT_RESOURCE_GROUP() |
++--------------------------+
+| rg1                      |
++--------------------------+
+1 row in set (0.00 sec)
+```
+
+Execute `SET RESOURCE GROUP` to set the resource group for the current session to `rg2` and view the resource group bound to the current user:
```
Comment (qiancai) [inline functions-and-operators/tidb-functions.md:374]:
```suggestion
Execute `SET RESOURCE GROUP` to set the resource group for the current session to `rg2`, and then view the resource group bound to the current user:
```

File: functions-and-operators/tidb-functions.md [hunk 8]
```diff
@@ -39,6 +39,7 @@ The following functions are TiDB extensions, and are not present in MySQL:
 | `VITESS_HASH(str)` | The `VITESS_HASH` function returns the hash of a string that is compatible with Vitess' `HASH` function. This is intended to help the data migration from Vitess. |
 | `TIDB_SHARD()` | The `TIDB_SHARD` function can be used to create a shard index to scatter the index hotspot. A shard index is an expression index with a `TIDB_SHARD` function as the prefix.|
 | `TIDB_ROW_CHECKSUM()` | The `TIDB_ROW_CHECKSUM` function is used to query the checksum value of a row. This function can only be used in `SELECT` statements within the FastPlan process. That is, you can query through statements like `SELECT TIDB_ROW_CHECKSUM() FROM t WHERE id = ?` or `SELECT TIDB_ROW_CHECKSUM() FROM t WHERE id IN (?, ?, ...)`. See also: [Data integrity validation for single-row data](https://docs.pingcap.com/tidb/stable/ticdc-integrity-check). |
+| `CURRENT_RESOURCE_GROUP()`  | The `CURRENT_RESOURCE_GROUP` function is used to return the resource group name that the current session is bound to. See also: [Use Resource Control to Achieve Resource Isolation](/tidb-resource-control.md) |
```
Comment (qiancai) [inline functions-and-operators/tidb-functions.md:42]:
```suggestion
| `CURRENT_RESOURCE_GROUP()`  | The `CURRENT_RESOURCE_GROUP` function is used to return the resource group name that the current session is bound to. See also: [Use Resource Control to Achieve Resource Isolation](/tidb-resource-control.md). |
```

Comment (qiancai) [inline functions-and-operators/tidb-functions.md:42]:
```suggestion
| `CURRENT_RESOURCE_GROUP()`  | The `CURRENT_RESOURCE_GROUP` function is used to return the resource group name that the current session is bound to. See also: [Use Resource Control to Achieve Resource Isolation](/tidb-resource-control.md). |
```

File: functions-and-operators/tidb-functions.md [hunk 9]
```diff
@@ -337,3 +339,50 @@ The output is as follows:
...
+
+### CURRENT_RESOURCE_GROUP
+
+The `CURRENT_RESOURCE_GROUP` function is used to show the resource group name that the current session is bound to. When the [Resource control](/tidb-resource-control.md) feature is enabled, the available resources that can be used by SQL statements are restricted by the resource quota of the bound resource group.
+
+When a session is established, TiDB binds the session to the resource group that the login user is bound to by default. If the user is not bound to any resource groups, the session is bound to the `default` resource group. Once the session is established, the bound resource group will not change by default, even if the bound resource group of the user is changed via [modifying the resource group bound to the user](/sql-statements/sql-statement-alter-user.md#modify-basic-user-information). To change the bound resource group of the current session, you can use [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md).
+
+#### Example
+
+Create a user `user1`, create two resource groups `rg1` and `rg2`, and bind the user `user1` to the resource group `rg1`:
```
Comment (hfxsd) [inline functions-and-operators/tidb-functions.md:351]: 下面的命令没有 `rg2`

Comment (qiancai) [reply functions-and-operators/tidb-functions.md:351]: @glorv PTAL

Comment (glorv) [reply functions-and-operators/tidb-functions.md:351]: fixed

**Pull request merged**